Header header

int32 left_segmentation
int32 right_segmentation
